Homework lesson 8 added.

iBubbleGun opened the pull request:
Added homework lesson 8.
Please, check it.

iBubbleGun committed:
Added homework lesson 8.

kartius commented on src/main/java/edu/hillel/homework/lesson8/calculators/ArrayValueCalculator.java:
```diff
@@ -0,0 +1,69 @@
+                        throw new ArrayDataException("An error occured!\n" +
+                                "Array element \"" + inputArray[i][j] + "\" " +
+                                "with index [" + i + "][" + j + "]" +
+                                " cannot be converted to a number because it's not a number!");
```
What I recommend here  - is to overload constructor of ArrayDataException with additional parameter Trowable and create ArrayDataException with message and Exception from catch block do not loose original stacktrace of exception

iBubbleGun replied:
The proposal has been implemented. Code updated. Check please. Now the original stack trace is not lost and can be displayed at the place where the method was used.

iBubbleGun committed:
Updated homework lesson 8.
Constructor ArrayDataException was overloaded with additional parameter Trowable. New ArrayDataException will be create with message and stack trace exception from catch block. So now we do not loose original stack trace of exception.

iBubbleGun committed:
Updated homework lesson 8.
